code/datalab-app/package.json

```diff
@@ -77,6 +79,12 @@
       "transform-object-rest-spread"
     ]
   },
+  "browserslist": [
```

Comment:
Could you see if it works without this? I'm not sure what it is for. Might be for eslint so maybe remove it locally, run yarn lint and see if that is successful. If not leave it in.
Might also be worth a google understand what this does.

Reply:
Seems to work without this. There are no more errors output by eslint when removing this (there are however 252 errors with or without this.)

Looks like it is configuration for this tool (https://github.com/browserslist/browserslist-example). Seems Babel uses it to work out what browser/node versions are being targeted and therefore what language features it is allowed to use during compilation. Not sure whether it is worth leaving in or not based on this information.